provider/bidengine/order.go

```diff
@@ -228,6 +267,8 @@ loop:
 			}
 
 			// Fulfillment placed.
+			o.bidPlaced = true
+			result.Value()
```

Comment:
I don't think it's needed

Reply:
Yeah the `result.Value()` isn't doing anything anymore. I'll kill it